.env.example

```diff
@@ -1,5 +1,7 @@
 # Environment variables
 # Copy to a file named .env and add your values
+FLASK_APP=verify
```

Comment:
this file isnt referenced in the README. the user should instructed to copy this file and update it with their information